src/main/src/main/java/org/geoserver/catalog/ResourcePool.java

```diff
@@ -2167,6 +2167,11 @@ public BufferedReader readStyle(StyleInfo style) throws IOException {
         if (styleResource == null) {
```

Comment:
This did not work anyway since: `assert r != null;` [here](https://github.com/geoserver/geoserver/blob/c3fc435088348df4096ed4dc00035c0994deaba0/src/main/src/main/java/org/geoserver/config/GeoServerDataDirectory.java#L670)